README.md

```diff
@@ -6,147 +6,127 @@
-6. Click Add Package. Do not forget "Add to Target".
-7. Choose Dependency Rule (e.g., "Up to Next Major Version" or "Exact Version").
+Select the **AtlasSupportSDK** from the search results, configure **Dependency Rule** and **Add to Project** settings, and click **Add Package** button.
 
 ### CocoaPods
 
-Add the SDK to your `Podfile`:
+In your project add **AtlasSupportSDK** dependency to the **Podfile**:
 
-```ruby
+```swift
 platform :ios, '13.0'
 use_frameworks!
 
 target 'YourApp' do
-  pod 'AtlasSupportSDK', '~> 0.0.6'
+  pod 'AtlasSupportSDK', '~> 0.1.2'
 end
 ```
 
-Run the following command in your terminal:
+Pull CocoaPods dependencies:
 
-```bash
+```
 pod install
 ```
 
-Open the `.xcworkspace` file in Xcode.
-
 ---
 
-## Usage
-
-### Import the SDK
+## Setup
 
-In any file where you want to use the SDK, import it:
+Import the package into your code:
 
 ```swift
 import AtlasSupportSDK
 ```
 
-In this example, we import the SDK into HomeViewController and TabBarController because these are the two navigation points leading to the AtlasViewController.
+Connect the SDK to your account:
 
----
+```swift
+AtlasSDK.setAppId("YOUR_APP_ID")
+```
 
-### Initialize and Configure the SDK
+---
 
-Call the appropriate initialization and configuration methods erly. Depends on your app architecture.
+## Identify your users
 
-For example:
+Make the following call with user details wherever a user logs into your application and you get the user details for the first time.
 
 ```swift
-AtlasSDK.setAppId("YOUR_APP_ID")
+AtlasSDK.identify(userId: "...",
+    userHash: nil,
+    name: "...",
+    email: "...",
+    phoneNumber: "...")
 ```
 
-You can find `YOUR_APP_ID` in the Atlas application: https://app.atlas.so/settings/company
+To make sure scammer can't spoof a user, you should pass a `userHash` into identify call (read more about it here).
 
----
+Unknown properties, like `name` or `phoneNumber` should be set to nil (not empty string) so they won't be overriden if they have previously been stored.
+
+When you want to update the user's details you can call `identify` method again.
+
+Additionally, there is a `logout` method available to clear the user's session when they log out of your application.
+
+```swift
+AtlasSDK.logout()
+```
 
-### Presenting the Atlas Chat ViewController
+## Show UI
 
-The `getAtlassViewController` method can return an optional value. This occurs if the appId is not provided, which is required for initializing the SDK correctly.
+To use Atlas UI you need to get instance of the view controller and insert it into your view:
 
 ```swift
 guard let atlassViewController = AtlasSDK.getAtlassViewController() else {
-    print("HomeViewController Error: Can not create AtlasSDK View Controller")
+    print("Can not create AtlasSDK View Controller")
     return
 }
   
-navigationController?.present(atlassViewController, animated: true)
+navigationController.present(atlassViewController, animated: true)
```

Comment:
Revert this change.

Reply:
We should not show that this is okay not to call `present()` method, or otherwise we need to show how to handle it.
Let's expect that developers know what to do if navigationController is not presented